lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2399,10 +2406,20 @@ LogicalResult StmtEmitter::visitStmt(InstanceOp op) {
 
     // Emit the value as an expression.
     ops.clear();
-    // output ports were lowered to wire
-    if (elt.isOutput())
+
+    // output ports that are not connected to single use output ports were lowered to wire
+    OutputOp output;
+    if (!elt.isOutput())
+      emitExpression(portVal, ops);
+    else if (portVal.hasOneUse() && (output = dyn_cast_or_null<OutputOp>(portVal.getUses().begin()->getOwner()))) {
```

Comment:
Plz fit to 80 columns (run clang format).  Also, please put braces around the previous "if" to be consistent with the else.